src/OpenVRTrackersDriver.cpp

```diff
@@ -105,6 +119,11 @@ bool openvr::DevicesManager::initialize()
 
     const auto lock = std::unique_lock(pImpl->mutex);
 
+    if(vrOrigin.compare("standing") == 0) {
+        pImpl->origin = TrackingUniverseOrigin::Standing;
```

Comment:
I would prefer if we set explicitly the origin every time, thus removing the initialization done in the constructor.

Reply:
Yes, done in https://github.com/ami-iit/yarp-openvr-trackers/pull/12/commits/2c1a682b998a0d629495c96f3ded5661eff20e4a